src/crypting/Main.java

```diff
@@ -0,0 +1,36 @@
+package crypting;
+
+import crypting.strategy.ShiftCipher;
+import crypting.strategy.UnicodeCipher;
+
+public class Main {
+
+    public static void main(String[] args) {
+        CliParser parser = new CliParser(args);
+        Cryptor cryptor;
+
+        String outFilePath = parser.optionOf("-out");
+        String inFilePath = parser.optionOf("-in");
+        String mode = parser.optionOf("-mode");
+        String algorithm = parser.optionOf("-alg");
+        String data = parser.optionOrDefault("-data", "");
+        int key = Integer.parseInt(parser.optionOrDefault("-key", "0"));
+
+        cryptor = new Cryptor("unicode".equals(algorithm) ?
+                new UnicodeCipher() :
+                new ShiftCipher()
+        );
+
+        if (data.isEmpty() && inFilePath != null)
+            data = FileReaderWriter.readFrom(parser.optionOf("-in"));
+
+        String cryptedData = "dec".equals(mode) ?
```

Comment:
What if I pass mode as `Dec`?

Reply:
Should I convert all arguments to lower case (or upper) or throw an exception in that case?
In another words should my program be registry sensible for arguments?

Reply:
> You can do whatever you want 👌

Ok, made registry sensible, so now arguments "Dec" and "dec" are different :)